appinventor/components/src/com/google/appinventor/components/runtime/TextBox.java

```diff
@@ -183,6 +187,24 @@ public void MultiLine(boolean multiLine) {
     view.setSingleLine(!multiLine);
   }
 
+  @SimpleProperty(
+    category = PropertyCategory.BEHAVIOR,
+    description = "Whether the %type% is read-only. By default, this is true."
```

Comment:
Don't you mean this property is false by default?

Reply:
@SusanRatiLane Yes! I will update that. Thanks!